example/index.html

```diff
@@ -26,9 +30,39 @@
     </style>
 </head>
 <body>
-<h1>OpenSea's Embeddable Nft!</h1>
+<h1 class="title header">OpenSea's Embeddable Nft!</h1>

```

Comment:
Let's change to "Embeddable NFTs from OpenSea" and add a subheader: "Show users any crypto collectible and let them buy it straight from your site, using a minimal-dependency web component!"